_config.yml

```diff
@@ -16,7 +16,6 @@ cache:
 # Redefine custom file paths. Introduced in NexT v6.0.2. If commented, will be used default custom file paths.
 # For example, you want to put your custom styles file outside theme directory in root `source/_data`, set `styles: source/_data/styles.styl`
 #custom_file_path:
-  # Default paths: layout/_custom/*
   #head: source/_data/head.swig
```

Comment:
If we uncomment all paths by default, it will no bugs later?

Reply:
I have removed `layout/_custom/*`, so it affects the people who modify those files.

If you only cancel the comment, it would works very well. Because it's compatible already by `scripts/filters/default-injects.js`.